site/app/controllers/student/TeamController.php

```diff
@@ -6,9 +6,18 @@
 use app\controllers\admin\AdminGradeableController;
 use app\libraries\FileUtils;
 use app\libraries\response\RedirectResponse;
+use app\models\Team;
 use Symfony\Component\Routing\Annotation\Route;
 
 class TeamController extends AbstractController {
+    /**
+     * Check if a string has special characters.
+     */
+    public static function hasSpecialCharacters(string $str): bool {
```

Comment:
Could you add two tests for this function to https://github.com/Submitty/Submitty/blob/main/site/tests/app/controllers/student/TeamControllerTester.php, one for a team name that has no special characters and one that does to validate behavior?

Reply:
sure
So, there's already a test named `testSetTeamName` to set team name which implicitly checks for the special characters as of now
Thus, I wrote only a single test `testSetTeamNameWithSpecialCharacters` to validate the behavior
You may review the code.